src/main/java/com/google/devtools/build/lib/sandbox/SandboxOptions.java

```diff
@@ -226,13 +226,22 @@ public ImmutableSet<Path> getInaccessiblePaths(FileSystem fs) {
   }
 
   @Option(
-      name = "experimental_enable_docker_sandbox",
+          name = "experimental_enable_docker_sandbox",
+          defaultValue = "false",
+          documentationCategory = OptionDocumentationCategory.EXECUTION_STRATEGY,
+          effectTags = {OptionEffectTag.EXECUTION},
+          help =
+                  "Enable Docker-based sandboxing. This option has no effect if Docker is not installed.")
+  public boolean enableDockerSandbox;
+
+  @Option(
+      name = "experimental_enable_docker_sandbox_symlinked_spawn",
       defaultValue = "false",
       documentationCategory = OptionDocumentationCategory.EXECUTION_STRATEGY,
       effectTags = {OptionEffectTag.EXECUTION},
       help =
-          "Enable Docker-based sandboxing. This option has no effect if Docker is not installed.")
-  public boolean enableDockerSandbox;
+          "When Docker-based sandboxing is enabled, use SymlinkedSandboxedSpawn instead of CopyingSandboxedSpawn")
```

Comment:
This should be written in a user-facing way - they don't (and shouldn't have to) know what these classes are :⁠-⁠)